bindings/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.12)
+cmake_policy(VERSION 3.12)
+
+project(PortAudio LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+# Todo (multi-generator): Add support for multiple generators like:
+# - {Debug, Release} x {Static, Dynamic} x {MT, MD (Windows only)}
+set(portaudio-cpp-sources)
+add_subdirectory(source/portaudiocpp) # define source-files
```

Comment:
IMO this indirection makes this harder to read for no clear benefit. I would prefer the list of source files to be in this file, not delegated to a subdirectory. You can simply list them in the `add_library` call below.

Reply:
Mhh, I've done this, to reduce the bloat in the main cmake file. For the understanding what this code does, it is not relevant, which source file is added, but it is required to add them manually. (To prevent [this](https://cmake.org/cmake/help/latest/command/file.html#glob-recurse)) 

it might be more readable to move the contents in a add_sources.cmake file, instead an subdirectory. Is this better for you, or should I still remove it?

Reply:
Moving this to a separate module wouldn't help anything.